staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -100,7 +100,15 @@ type VirtualMachineInstanceSpec struct {
 	SchedulerName string `json:"schedulerName,omitempty"`
 	// If toleration is specified, obey all the toleration rules.
 	Tolerations []k8sv1.Toleration `json:"tolerations,omitempty"`
-
+	// TopologySpreadConstraints describes how a group of VMIs ought to spread across topology
```

Comment:
VMIs ought to spread acros -> VMIs will be spread  (I am not a native speaker so maybe this is correctly used)

Reply:
I think it's a less common way of saying "should"  :) I changed it to "will be spread across"